### PR TITLE
Fix checks and apply suggested changes for search

### PR DESCRIPTION
**Pull request by josueetcom**
You can see the changes in action at https://sps-travelbud.appspot.com/

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
   },
   "extends": ["eslint:recommended", "google"],
   "rules": {
-    "no-multi-spaces": "off"
+    "no-multi-spaces": "off",
+    "require-jsdoc": "off",
+    "valid-jsdoc": "off"
```
Comment (josueetcom) [inline .eslintrc.json:13]: This ensures JSDocs are no longer required for javascript

File: makefile [hunk 2]
```diff
@@ -9,7 +9,6 @@ node_modules:
 
 pretty: node_modules
 	$(PRETTIER) --write src/main/webapp/*.{html,css}
-	$(ESLINT) --fix src/main/webapp/*.js
```
Comment (josueetcom) [inline makefile:12]: this ensures files are still formatted, even if there are lint issues

**Review by NzingaHolloman — APPROVED** [review]
lgtm

**Review by TengkuHanaShafikah — APPROVED** [review]
These changes look great! Thank you so much for your help, Josue!

**Pull request merged**